scripts/generate_cc.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# ============================================================================ #
+# Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+# Usage:
+# bash scripts/generate_cc.sh -c
+# -or-
+# bash scripts/build_cudaq.sh -p
+# -or-
+# bash scripts/build_cudaq.sh -c -p
```

Comment:
Perhaps a copy/paste error?
```suggestion
# bash scripts/generate_cc.sh -p
# -or-
# bash scripts/generate_cc.sh -c -p
```

Reply:
Thanks, adding the -c and -p options at the same time means generating both C code coverage and python code coverage.

Reply:
> Thanks, adding the -c and -p options at the same time means generating both C code coverage and python code coverage.

I was meaning that I wonder if the references to `build_cudaq.sh` are correct. Shouldn't they be referencing `generate_cc.sh`?

Reply:
Sorry, I didn't catch your meaning correctly :)